ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.html

```diff
@@ -2,10 +2,11 @@
     <section class="introduction">
         <h2 class="title" translate>App.HomePage.IntroductionSection.Title</h2>
         <p class="text" translate>App.HomePage.IntroductionSection.Text</p>
-        <a mat-raised-button
+        <a mat-flat-button
            [routerLink]="'/' + AppRoutes.CountMeIn"
            class="btn join-cmi-btn center"
-           [innerText]="'App.HomePage.JoinCountMeInButton' | translate">
```

Comment:
let's be careful about innerText/innerHTML. It may override material directive html

Reply:
you mean that if we have some html in a translation then using `[innerText]`  will skip the html ?

Reply:
I mean that if material component/directive adds custom html we overwrite it with innerText/innerHTML

Reply:
that might lead to the broken material behavior, for example removing some elements which are responsible for focus styles

Reply:
got it, thanks.